projects/distributed-tracing/src/shared/dashboard/widgets/waterfall/waterfall/waterfall-chart.component.ts

```diff
@@ -202,7 +202,7 @@ export class WaterfallChartComponent implements OnChanges {
 
     let markerData: MarkerTooltipData = {
       relativeTimes: [],
-      summary: spanWaterfallData.logEvents[0].summary
+      summary: spanWaterfallData.logEvents.find(logEvent => logEvent.timestamp === marker.timestamps[0])?.summary ?? ''
```

Comment:
Could you please explain the bugs in the description or add any github issue if present?

Reply:
Done!!